Backend/app/rider-platform/rider-app/Main/src/Domain/Action/UI/Cancel.hs

```diff
@@ -156,3 +168,25 @@ cancelSearch personId dcr = do
         Esq.runTransaction $ QPFS.updateStatus personId DPFS.IDLE
         QEstimate.updateStatus dcr.estimateId DEstimate.CANCELLED
   QPFS.clearCache personId
+
+getDistanceBetweenPoints ::
+  ( EncFlow m r,
+    CacheFlow m r,
+    EsqDBFlow m r,
+    Metrics.CoreMetrics m,
+    Maps.HasCoordinates a,
+    Maps.HasCoordinates b
+  ) =>
+  Id Merchant ->
+  a ->
+  b ->
+  m Meters
+getDistanceBetweenPoints merchantId a b = do
```

Comment:
don't use variable name as `a` and `b`. Rename them properly.

Reply:
pushed new changes.

Reply:
pushed new changes.